change/@microsoft-teams-js-a5218507-221a-43fb-9564-a80174a8f210.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Added optional field `messageId` to `stageView.open`. messageId points to the root message of the channel meeting. It's a unique identifier of the meeting on that channel.",
```

Comment:
Thank you for updating this, but what does setting `messageId` actually do? Does it somehow alter the functionality of the stageView that gets opened?

Reply:
Its actually needed to make sure that we open the stage in the correct channel meeting. messageId points to the root of that channel meeting